wsbroadcastserver/wsbroadcastserver.go

```diff
@@ -169,8 +169,9 @@ func (s *WSBroadcastServer) StartWithHeader(ctx context.Context, header ws.Hands
 
 		safeConn := deadliner{conn, s.config().IOTimeout}
 
+		// Set requestedSeqNum to max if client doesn't provide it
+		requestedSeqNum := arbutil.MessageIndex(^uint64(0))
 		var feedClientVersionSeen bool
-		var requestedSeqNum arbutil.MessageIndex
```

Comment:
Do we still support an old nitro version that wouldn't send the header and get hit by this? Why was this change made?

Reply:
The requested sequence number was added during beta, and we have had several consensus versions since then.  So guaranteed all clients will be sending this.

It was removed to reduce data sent if someone simply connects to port manually or with a script.